symcrypt-sys/build.rs

```diff
@@ -48,13 +48,15 @@ fn main() {
         // HMAC FUNCTIONS
         .allowlist_function("^(SymCryptHmacSha256.*)$")
         .allowlist_function("^(SymCryptHmacSha384.*)$")
-        // GCM FUNCTIONS
+        // CM FUNCTIONS
```

Comment:
?

Reply:
Thought I needed to support other CM functions, will change back to GCM.